PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -583,7 +583,7 @@ def _addUTagToTaus(process, task,
     pp_on_AA.toModify(_uTagToTaus_switches, storePNetCHSjets=False, storeUParTPUPPIjets=False)
     _addUTagToTaus(process, task,
                    storePNetCHSjets = _uTagToTaus_switches.storePNetCHSjets.value(),
-                   storeUParTPUPPIjets = _uTagToTaus_switches.storePNetCHSjets.value(),
+                   storeUParTPUPPIjets = _uTagToTaus_switches.storeUParTPUPPIjets.value(),
```

Comment:
It would be good to backport this fix to 15_0_X too. 
At the moment it does not affect anything as `storePNetCHSjets` and `storeUParTPUPPIjets` are always both `True` or both `False`, but it's always better to have the correct implementation to avoid any future surprises.

Reply:
Good point, @y19y19 can you make a backport of just this change?
(I noticed this while we were reviewing the relevant sequences and decided it was easiest just to fix it here...)

Reply:
Backport PR is commited. #47715

Reply:
Thanks a lot!